run-build-functions.sh

```diff
@@ -626,22 +626,22 @@ install_dependencies() {
 
     restore_home_cache ".node/corepack" "corepack dependencies"
 
-		if has_feature_flag "$featureFlags" "build-image_use_new_package_manager_detection"; then
-			local pkgManager=$(echo "$buildInfo" | jq -r '.packageManager.name')
-			case $pkgManager in
-				"yarn")
-					run_yarn "$YARN_VERSION" "$featureFlags"
-					;;
-				"pnpm")
-					run_pnpm "$PNPM_VERSION" "$featureFlags"
-					;;
-				*)
-					run_npm "$featureFlags"
-					;;
-			esac
-
-		else
-			# feature flag turned off use the old detection
```

Comment:
This still looks off(?) the `if` feature flag check should be under `if [ -f package.json ]` right?

Reply:
should be now fixed